README.md

```diff
@@ -1 +1,12 @@
+[![Build Status](https://travis-ci.org/andela-gike/Checkpoint-1.svg?branch=master)](https://travis-ci.org/andela-gike/Checkpoint-1)
+[![Coverage Status](https://coveralls.io/repos/github/andela-gike/Checkpoint-1/badge.svg?branch=master)](https://coveralls.io/github/andela-gike/Checkpoint-1?branch=master)
```

Comment:
You also need to specify the task for Travis to do after running your tests (i.e send code coverage report to Coveralls or CodeClimate) in your travis.yml file else your coverage badge will remain unknown.